ted_sws/data_manager/adapters/mapping_suite_repository.py

```diff
@@ -12,6 +12,7 @@
     SPARQLTestSuite, MetadataConstraints, TransformationTestData, ConceptualMapping
 from ted_sws.data_manager.adapters import inject_date_string_fields, remove_date_string_fields
 from ted_sws.data_manager.adapters.repository_abc import MappingSuiteRepositoryABC
+from ted_sws.data_manager.services.mapping_suite_resource_manager import read_flat_file_resources
```

Comment:
This import is invalid in clean architecture, this import brings dependence in adapter from service, this is very bad dependence.

Reply:
yes. probably a ghost dependency from start of the branch. will move it.